common/shlibs

```diff
@@ -3318,28 +3318,28 @@ libarcan_a12.so.0.1 arcan-0.6.1.1_1
 liblwipv6.so.2 lwipv6-1.5a_1
 libpipewire-0.3.so.0 libpipewire-0.3.6_1
 libvolk.so.3.1 volk-3.1.0_1
-libgnuradio-runtime.so.3.10.10 gnuradio-3.10.10.0_1
-libgnuradio-pmt.so.3.10.10 gnuradio-3.10.10.0_1
-libgnuradio-blocks.so.3.10.10 gnuradio-3.10.10.0_1
-libgnuradio-audio.so.3.10.10 gnuradio-3.10.10.0_1
-libgnuradio-fec.so.3.10.10 gnuradio-3.10.10.0_1
-libgnuradio-fft.so.3.10.10 gnuradio-3.10.10.0_1
-libgnuradio-filter.so.3.10.10 gnuradio-3.10.10.0_1
-libgnuradio-analog.so.3.10.10 gnuradio-3.10.10.0_1
-libgnuradio-digital.so.3.10.10 gnuradio-3.10.10.0_1
-libgnuradio-dtv.so.3.10.10 gnuradio-3.10.10.0_1
-libgnuradio-channels.so.3.10.10 gnuradio-3.10.10.0_1
-libgnuradio-trellis.so.3.10.10 gnuradio-3.10.10.0_1
-libgnuradio-video-sdl.so.3.10.10 gnuradio-3.10.10.0_1
-libgnuradio-vocoder.so.3.10.10 gnuradio-3.10.10.0_1
-libgnuradio-wavelet.so.3.10.10 gnuradio-3.10.10.0_1
-libgnuradio-uhd.so.3.10.10 gnuradio-3.10.10.0_1
-libgnuradio-network.so.3.10.10 gnuradio-3.10.10.0_1
-libgnuradio-soapy.so.3.10.10 gnuradio-3.10.10.0_1
-libgnuradio-qtgui.so.3.10.10 gnuradio-3.10.10.0_1
-libgnuradio-zeromq.so.3.10.10 gnuradio-3.10.10.0_1
-libgnuradio-iio.so.3.10.10 gnuradio-3.10.10.0_1
-libgnuradio-pdu.so.3.10.10 gnuradio-3.10.10.0_1
+libgnuradio-runtime.so.3.10.11 gnuradio-3.10.11.0_2
+libgnuradio-pmt.so.3.10.11 gnuradio-3.10.11.0_2
+libgnuradio-blocks.so.3.10.11 gnuradio-3.10.11.0_2
+libgnuradio-audio.so.3.10.11 gnuradio-3.10.11.0_2
+libgnuradio-fec.so.3.10.11 gnuradio-3.10.11.0_2
+libgnuradio-fft.so.3.10.11 gnuradio-3.10.11.0_2
+libgnuradio-filter.so.3.10.11 gnuradio-3.10.11.0_2
+libgnuradio-analog.so.3.10.11 gnuradio-3.10.11.0_2
+libgnuradio-digital.so.3.10.11 gnuradio-3.10.11.0_2
+libgnuradio-dtv.so.3.10.11 gnuradio-3.10.11.0_2
+libgnuradio-channels.so.3.10.11 gnuradio-3.10.11.0_2
+libgnuradio-trellis.so.3.10.11 gnuradio-3.10.11.0_2
+libgnuradio-video-sdl.so.3.10.11 gnuradio-3.10.11.0_2
+libgnuradio-vocoder.so.3.10.11 gnuradio-3.10.11.0_2
+libgnuradio-wavelet.so.3.10.11 gnuradio-3.10.11.0_2
+libgnuradio-uhd.so.3.10.11 gnuradio-3.10.11.0_2
+libgnuradio-network.so.3.10.11 gnuradio-3.10.11.0_2
+libgnuradio-soapy.so.3.10.11 gnuradio-3.10.11.0_2
+libgnuradio-qtgui.so.3.10.11 gnuradio-3.10.11.0_2
+libgnuradio-zeromq.so.3.10.11 gnuradio-3.10.11.0_2
+libgnuradio-iio.so.3.10.11 gnuradio-3.10.11.0_2
+libgnuradio-pdu.so.3.10.11 gnuradio-3.10.11.0_2
```

Comment:
because these changed, every package that uses them should have the revision incremented. you can do that with xtools with `xrevshlib gnuradio | xargs xrevbump 'rebuild for gnuradio-3.10.11.0'`

also, as the lint says, gnuradio's revision should be set to 1 here too (`_2` -> `_1`)

Reply:
Got it. I'll try and knock that out this evening. Thanks for the input!

Reply:
okay, that should be sorted. Should I be rebuilding and testing gqrx and the other packages affected by me updating gnuradio?

Reply:
yes. also you still need to change the revision in `common/shlibs`

Reply:
Sorry had some stuff going on in my personal life. gqrx and gnuradio-osmosdr are easy for me to test, but I don't have the hardware for the others unforntunately.